PocketKit/Sources/PocketKit/Article/ReadableViewModel/RecommendationViewModel.swift

```diff
@@ -186,7 +215,7 @@ extension RecommendationViewModel {
         selectedRecommendationToReport = recommendation
     }
 
-    private func favorite() {
+    internal func favorite() {
```

Comment:
```suggestion
    func favorite() {
```

Internal by default; no need to explicitly mark it as such.